spec/lib/integration/integration_spec.rb

```diff
@@ -1,23 +1,37 @@
 require 'spec_helper'
+
 require 'capybara_discoball'
-require 'derby/server'
+require 'lamprey'
 
 describe 'Integration tests' do
-  let!(:derby_server) do
-    Capybara::Discoball::Runner.new(Derby::Server).boot
+  before(:all) do
+    WebMock.disable!
+  end
+
+  after(:all) do
+    WebMock.enable!
+  end
+
+  let!(:ldp_server) do
+    Capybara::Discoball::Runner.new(RDF::Lamprey).boot
   end
 
   let(:debug) { ENV.fetch('DEBUG', false) }
 
   let(:client) do
-    Faraday.new(url: derby_server) do |faraday|
+    Faraday.new(url: ldp_server) do |faraday|
       faraday.response :logger if debug
       faraday.adapter Faraday.default_adapter
     end
   end
 
   subject { Ldp::Client.new client }
 
+  before do
+    # Initialize LDP server
+    subject.put "/", ""
+  end
```

Comment:
I'm not sure why this is needed with Lamprey, but without it the first test will fail unless that test is "creating resources".